src/bp-templates/bp-nouveau/includes/activity/ajax.php

```diff
@@ -370,8 +370,10 @@ function bp_nouveau_ajax_new_activity_comment() {
 	if ( ! is_user_logged_in() ) {
 		wp_send_json_error( $response );
 	}
+	
+	$content = apply_filters( 'bb_is_activity_content_empty', $_POST );
```

Comment:
I don't think we need to add &nbsp; here for activity if it contains gif only. we don't add &nbsp; for media and document when they added in activity if activity has no content in it. please remove this.